sdk/appconfiguration/ci.yml

```diff
@@ -17,6 +17,13 @@ pr:
     include:
     - sdk/appconfiguration/
 
+resources:
```

Comment:
This is an unexpected side-effect and I really wish that we could define these in some common manner, but I don't know of a way right now.

@mitchdenny what is your thought about specifying a ref here as well?

Reply:
Unfortunately resources cannot be added to templates.  I couldn't figure out a way to keep it in a single common place. Still it might not be a bad idea if many tools will be continuously loaded from azure -sdk - tools repo

Reply:
I couldn't find another way either and I agree this isn't necessary ideal but isn't a blocker either.